kitchen/bin/waiter-init

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# A wrapper script for the Waiter-specific setup for a user command in a Waiter-K8s pod.
+# The script is usually invoked by prepending it to the user's Waiter command.
+#
+# A single argument is expected, the user's command string,
+# which is executed as its own bash shell process.
+
+# Ignore SIGTERM sent by Kubernetes on pod deletion,
+# waiting for SIGKILL (force delete) before exiting.
+trap : SIGTERM
+
+# Run the user's Waiter app command,
+# copying stdout and stderr to respectively named files.
+# We tee the output so that stdout and stderr are still accessible
+# via the Kubernetes `kubectl logs <pod-name>` command.
+/bin/bash -c "$1" 1> >(tee stdout) 2> >(tee stderr 1>&2)
```

Comment:
On a related note, should we consider supporting `WAITER_SANDBOX` (while temporarily keeping support for `MESOS_SANDBOX`)?

Reply:
Setting `WAITER_SANDBOX` for Waiter on Kubernetes is easy (we'd just do it the same way we're currently setting `MESOS_SANDBOX`).

My impression was that injecting such environment variables into the Mesos containers in Waiter on Marathon was the problem (because we don't have the value until after the container is already created), and it wouldn't make sense to run this script in a Mesos container.

Reply:
Digression: What are the reservations for running a similar script around Mesos tasks?

Reply:
> Digression: What are the reservations for running a similar script around Mesos tasks?

There's no problem with auto-injecting a *similar* script into Marathon commands. Just not this script, as it's tailored for Kubernetes pods.